pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -342,33 +342,37 @@ func validateInterfaceNetworkBasics(field *k8sfield.Path, networkExists bool, id
 }
 
 func validateDHCPExtraOptions(field *k8sfield.Path, iface v1.Interface) (causes []metav1.StatusCause, done bool) {
```

Comment:
Please consider removing the named return values.
They were supposed to help when using [naked returns](https://go.dev/tour/basics/7), but IMO they don't improve readability in this case.
Same bellow.

Reply:
I added a new commit that does it